api/models/network/updateNetwork.js

```diff
@@ -0,0 +1,62 @@
+import { isEqual, difference, values, some } from 'lodash'
+import setupNetworkAttrs from './setupNetworkAttrs'
+
+export default function updateNetwork (userId, id, params) {
+  if (!userId) throw new Error('updateNetwork called with no userID')
+  if (!id) throw new Error('updateNetwork called with no ID')
+  return setupNetworkAttrs(userId, params).then(attrs =>
+    bookshelf.transaction(transacting =>
+      // NOTE: EnsureLoad not built to work with belongsToMany relations
```

Comment:
it's present in `afterUpdatingPost` because that function is being passed an instance, not an ID. So it could be useful in `afterUpdatingNetwork` as well, but not in `updateNetwork`. Perhaps that's what you were intending, but I was responding to the placement of the comment.